src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -679,6 +690,15 @@ export default function EarnPoolInfoScreen({ route, navigation }: Props) {
         linkUrl={dataProps.manageUrl}
         testId="DailyYieldRateInfoBottomSheet"
       />
+      <InfoBottomSheet
+        infoBottomSheetRef={safetyScoreInfoBottomSheetRef}
+        titleKey="earnFlow.poolInfoScreen.infoBottomSheet.safetyScoreTitle"
+        descriptionKey="earnFlow.poolInfoScreen.infoBottomSheet.safetyScoreDescription"
+        providerName={appName}
+        linkKey="earnFlow.poolInfoScreen.infoBottomSheet.safetyScoreRateLink"
```

Comment:
It looks like this text can fail to wrap, but this appears to be an existing issue with the `InfoBottomSheet` component.

<img src="https://github.com/user-attachments/assets/aa8fa096-5666-4ae6-903d-17344cbf6cfe" width="50%" height="auto" />

Reply:
Looks like this wraps correctly on large fonts so this seems like an edge case.